README.md

```diff
@@ -79,6 +79,19 @@ config :mix_test_watch,
   clear: true
 ```
 
+## Excluding files or directories
+
+To ignore changes from specific files or directories just add `exclude:` patterns
+to your config in `mix.exs`:
+
+```elixir
+config :mix_test_watch,
+  exclude: ["db_migration/.*", "useless_.*\.exs"]
+```
+
+It accepts either regexp struct `~r/\w+/` or pattern as string `"\w+"` which
```

Comment:
I think this would be fine if strings supported globs, but here we're just casting them to regex.